collector/logical_router_port_collector.go

```diff
@@ -0,0 +1,174 @@
+package collector
+
+import (
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/prometheus/client_golang/prometheus"
+	nsxt "github.com/vmware/go-vmware-nsxt"
+	"github.com/vmware/go-vmware-nsxt/manager"
+)
+
+func init() {
+	registerCollector("logical_router_port", newLogicalRouterPortCollector)
+}
+
+type logicalRouterPortCollector struct {
+	client *nsxt.APIClient
+	logger log.Logger
+
+	rxTotalPacketDesc   *prometheus.Desc
+	rxDroppedPacketDesc *prometheus.Desc
+	rxTotalByteDesc     *prometheus.Desc
```

Comment:
do we need include rxDroppedByte and txDroppedByte as well or the nsx client doesn't support this?

Reply:
There's no dropped_bytes metric in nsx client and api